index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
```

Comment:
- [ ] To ensure that your nav follows best practices and the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=mnMnPg6Wment6Asj-0), please adjust the margin and padding values of your nav element. You can use the following code as an example:

### Your design:
![image](https://user-images.githubusercontent.com/107952724/235244462-2ae9a855-e6a3-46ae-9154-02f9b82bc802.png)

### Figma design:
![image](https://user-images.githubusercontent.com/107952724/235244692-4dc6da26-1571-4c50-bbd0-2c3ec6cfa291.png)